releasenotes/notes/support-host-etc-folder-a45dc78da37872e1.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    When running inside a container with the host `/etc` folder mounted to `/host/etc`, the agent will now report the
+    kernel of the host instead of the one from the container.
```

Comment:
This doesn't sound right -- a container is running in namespaces on the host, so there aren't two kernels to report.  Is there a more precise way to phrase this?

Reply:
+1 to this. The comment in `pkg/config/config.go` also needs updating as well to match.

Reply:
I updated the wording toward  "distro informations"